packages/bell/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "",
   "scripts": {
     "build": "rollup -c",
-    "build:watch": "rollup -c --watch",
+    "build:watch": "rollup -cw",
```

Comment:
Is this correct? Looks like it was the previous version :P What does the -c does?